web/src/reload.ts

```diff
@@ -21,7 +20,16 @@ import * as util from "./util.ts";
 
 let server_reachable_check_failures = 0;
 
-const token_metadata_schema = z.object({url: z.string(), timestamp: z.number()});
+export const reload_metadata_schema = z.object({
+    reload_data: z.object({
+        hash: z.string(),
+        message_view_pointer: z.optional(z.number()),
+        message_view_scroll_offset: z.optional(z.number()),
+        compose_active_draft_id: z.optional(z.string()),
+        send_draft_immediately: z.optional(z.boolean()),
```

Comment:
Looking at these together, `compose_active_draft_send_immediately` might be a better name, to make it more obviously connected to the draft ID if more items enter the picture later.